ui/src/main/webapp/lib/static-site-generator/html.js

```diff
@@ -48,7 +48,7 @@ export default ({ assets, children }) => {
     <html>
       <head>
         <title>Admin Console</title>
-        <meta charset='UTF-8' />
+        <meta charSet='UTF-8' />
```

Comment:
❓ Why?
From [section 4.2.5.1 of the HTML5 spec](https://www.w3.org/TR/html5/document-metadata.html#standard-metadata-names): "Names are case-insensitive, and must be compared in an ASCII case-insensitive manner."

Reply:
This is a react quirk. What you are looking at is JSX not HTML which has a few slight naming changes. When it gets rendered it will be `charset`.